prow/cmd/hook/main.go

```diff
@@ -79,6 +82,7 @@ func gatherOptions() options {
 	flag.StringVar(&o.pluginConfig, "plugin-config", "/etc/plugins/plugins", "Path to plugin config file.")
 
 	flag.BoolVar(&o.dryRun, "dry-run", true, "Dry run for testing. Uses API tokens but does not mutate.")
+	flag.DurationVar(&o.gracePeriod, "grace-period", "180s", "On shutdown, try to handle remaining events for the specified duration. ")
```

Comment:
default is `180 * time.Second` here, and when people use the flag it becomes `--grace-period=180s`

Reply:
yep, you supply the parsed defaults, not the command line equivalent, the current value should be invalid.